pkg-manager/resolve-dependencies/src/resolveDependencies.ts

```diff
@@ -409,8 +414,24 @@ async function resolveDependenciesOfImporters (
       const postponedPeersResolutionQueue: PostponedPeersResolutionFunction[] = []
       const pkgAddresses: PkgAddress[] = []
 
-      const resolvedDependenciesOfImporter = await Promise.all(
-        extendedWantedDeps.map((extendedWantedDep) => resolveDependenciesOfDependency(
+      async function resolveDependenciesOfImporter (extendedWantedDep: ExtendedWantedDependency) {
```

Comment:
I don't usually nest functions. That makes it clear what is actually used by the function as everything needs to be passed through arguments. I suggest to move this function out.

Reply:
Got it. Thanks for explaining. I'll avoid nesting functions in future PRs now that I know the reason. 🙂